### PR TITLE
Update tuto erc20 to email to match new version of service erc20

### PR DESCRIPTION
**Pull request by NicolasMahe**
Update tuto erc20 to email to match new version of service erc20

Related to https://github.com/mesg-foundation/service-ethereum-erc20/pull/7

### CODE REVIEW

**Comment by antho1404** [comment]
Don't you think we should listen for a specific ERC20 that is done by the application ?

**Comment by NicolasMahe** [comment]
Yes we should, otherwise there are really 100 emails per blocks that will be send

**Comment by NicolasMahe** [comment]
Some links will appear broken in the preview because they are pointing to the master. they will be fixed when merged.

**Comment by NicolasMahe** [comment]
@mesg-foundation/core please review

**Comment by NicolasMahe** [comment]
@mesg-foundation/core ready to review

File: docs/tutorials/erc20-transfer-notifications/send-email-with-sendgrid/test.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "from": "sender@domain.tld",
-  "to": "__YOUR_EMAIL__",
+  "to": "nicolas@mahe.me",
```
[the remark this answers is not in the document]
Comment (NicolasMahe) [reply docs/tutorials/erc20-transfer-notifications/send-email-with-sendgrid/test.json]: I change it but was not able to edit my commits. Need to squash on merge.

**Comment by NicolasMahe** [comment]
**Need to squash on merge**

**Pull request merged**